From 8009c621ffd1ec504a6d5bffb236289d406a839a Mon Sep 17 00:00:00 2001
From: Kent McLeod <Kent.Mcleod@data61.csiro.au>
Date: Tue, 12 Sep 2017 13:58:54 +1000
Subject: [PATCH 4/5] Change irq throttling to 0

---
 sys/dev/pci/if_wm.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sys/dev/pci/if_wm.c b/sys/dev/pci/if_wm.c
index d075d673..2bc24fcd 100644
--- a/sys/dev/pci/if_wm.c
+++ b/sys/dev/pci/if_wm.c
@@ -4578,7 +4578,7 @@ wm_init_locked(struct ifnet *ifp)
 		 * divided by 4 to get "simple timer" behavior.
 		 */
 
-		sc->sc_itr = 1500;		/* 2604 ints/sec */
+		sc->sc_itr = 0;		/* No interrupt throttling */
 	}
 
 	error = wm_init_txrx_queues(sc);
-- 
2.11.0

